mqtt/mqtt-edgehub/src/auth/authorization/edgehub.rs

```diff
@@ -11,6 +11,7 @@ use mqtt_broker::{
 #[derive(Debug, Default)]
 pub struct EdgeHubAuthorizer {
     iothub_allowed_topics: RefCell<HashMap<ClientId, Vec<String>>>,
+    service_identities_cache: HashMap<String, ServiceIdentity>,
```

Comment:
@dmolokanov can you please review this file to make sure we are not breaking anything.